.github/workflows/release.yml

```diff
@@ -0,0 +1,112 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - ci/*
+      - release/*
```

Comment:
Shouldn't we only look at tags? Really, don't we trigger on release?

Reply:
Following the thought from https://github.com/filecoin-project/lotus/pull/11785#discussion_r1541430018, the tag push would trigger a proper release while a push to either ci/* or release/* branch would trigger a dry run release to see if everything's OK.

Reply:
Can we trigger the full run on github releases only (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release) to prevent accidental tag pushes from turning into releases.

Reply:
If you don't mind, I'd prefer to keep the triggers consistent with CircleCI during the trial period. This will make it easier for us to verify whether we're ready to deprecate CircleCI. Once we do that, I totally agree that it'd be a good idea to explore how we could improve and secure the release process further. Would that be alright?

And yes, we could definitely explore using the release trigger instead of the tag one in the future. It might need a slight rework of the workflow and/or underlying scripts since the context would be slightly different, but it's not an issue.

Reply:
Sg.